### ISSUE TITLE
Disabled three-state is enabled in normal Logisim.

### ISSUE THREAD

**Issue by htu86** [logisim-evolution/logisim-evolution#1901]
I've been using Logisim evolution to create some simple truth table calculators and when I disable the three-state property in the inputs, save it and open the same project in the old and official version of Logisim on windows, it resets the three-state + the text that is placed by outputs on the right is moved to the left. Is there any workaround for this? I am on MacOS and work with an Apple Sillicon laptop.

**Comment by maehne** [comment]
Yes, stop using the old Logisim, which development has been stopped years ago and migrate fully to Logisim-evolution. ;-) Being able to open old Logisim circuits after so many years still in Logisim-evolution is one thing. We don't try to break this without any good reason. However, guaranteeing backwards compatibility in the sense that you can open circuits created in Logisim-evolution also in the old Logisim version is a non goal due to our limited resources. We also don't see a real user benefit. Therefore, I am going to close this issue as won't fix. Nevertheless, feel free to reopen if you want to discuss this further or are willing to actively work on this issue. PRs are always welcome, as long as they don't negatively impact other functionality.
